### PR TITLE
Update NextCloud docs

### PR DESCRIPTION
**Pull request by J0WI**
Merging recent changes from https://github.com/nextcloud/docker/commits/master/README.md
Should be merged with https://github.com/docker-library/official-images/pull/5824

cc @tilosp

### CODE REVIEW

**Comment by tilosp** [comment]
The changes to `README.md `should not be committed according to the  [README](https://github.com/docker-library/docs#how-do-i-update-an-images-docs).

Otherwise it looks fine to me.

**Comment by J0WI** [comment]
@tianon these are the docs for https://github.com/docker-library/official-images/pull/5824, would be great to have them merged too :smiley:

File: nextcloud/content.md [hunk 1]
```diff
@@ -229,6 +257,16 @@ services:
 
 Then run `docker-compose up -d`, now you can access Nextcloud at http://localhost:8080/ from your host system.
 
+# Docker Secrets
+
+As an alternative to passing sensitive information via environment variables, _FILE may be appended to the previously listed environment variables, causing the initialization script to load the values for those variables from files present in the container. In particular, this can be used to load passwords from Docker secrets stored in /run/secrets/<secret_name> files. For example:
```
Comment (tianon) [inline nextcloud/content.md]: This functionality was reverted, wasn't it?

Comment (J0WI) [reply nextcloud/content.md]: Yes, I'll (re)implement this but I'd like to wait for https://github.com/docker-library/official-images/pull/5839.

Comment (J0WI) [reply nextcloud/content.md]: Removed this section for now.

**Comment by tianon** [comment]
Verified that https://github.com/docker-library/docs/compare/1022acbeb251020b2a0c395a8f7cf84d85c27690..d1809838a5819c54e85ee8d8b93ebbe34c5a2837 is the only content change since @tilosp said it looked good (removing the reverted `_FILE` bits), so I'm going to go ahead with the merge here. :+1:

(Of course, please holler if this was missing something. :+1: :heart:)

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**